roles/postgresql/tasks/main.yml

```diff
@@ -1,3 +1,31 @@
 ---
-- include_role:
-    name: galaxy/ANXS.postgresql
+- name: Install PostgreSQL
+  become: true
+  apt:
+    name:
+      - postgresql
+      - python-psycopg2
+
+- become: true
+  become_user: postgres
+  block:
+    - name: Create PostgreSQL database
+      postgresql_db:
+        name: "{{ database_name }}"
+
+    - name: Create PostgreSQL users
+      postgresql_user:
+        name: "{{ database_user }}"
+        password: "{{ database_password }}"
+        encrypted: false
```

Comment:
I left this in because it was in the config before, but I'm not sure that it's a good idea.

Reply:
Potential security issue indeed 🤔